### PR TITLE
Debugger 

### PR DESCRIPTION
**Pull request by martin-grofcik**

### CODE REVIEW

File: modules/flowable-ui-task/flowable-ui-task-rest/src/main/java/org/flowable/app/rest/runtime/RuntimeDisplayJsonClientResource.java [hunk 1]
```diff
@@ -128,7 +128,7 @@ public JsonNode getModelJSON(@PathVariable String processInstanceId) {
 
         ProcessInstance processInstance = runtimeService.createProcessInstanceQuery().processInstanceId(processInstanceId).singleResult();
         if (processInstance == null) {
-            throw new BadRequestException("No process instance found with id " + processInstanceId);
+            return getModelHistoryJSON(processInstanceId);
```
Comment (tijsrademakers) [inline modules/flowable-ui-task/flowable-ui-task-rest/src/main/java/org/flowable/app/rest/runtime/RuntimeDisplayJsonClientResource.java]: Hi Martin, this change modifies the behaviour of the getModelJson method. If you need the history json then the history endpoint should be invoked.

File: modules/flowable-ui-task/flowable-ui-task-app/src/main/resources/META-INF/flowable-app/flowable-app.properties [hunk 2]
```diff
@@ -96,7 +96,7 @@ idm.admin.password=test
 #app.redirect.url.on.authsuccess=http://localhost:8080/flowable-task
 #deployment.api.url=http://localhost:8080/flowable-task/process-api
 
-debugger.enabled=false
+debugger.enabled=true
```
Comment (tijsrademakers) [inline modules/flowable-ui-task/flowable-ui-task-app/src/main/resources/META-INF/flowable-app/flowable-app.properties]: Can you change this to false as the default?

Comment (martin-grofcik) [reply modules/flowable-ui-task/flowable-ui-task-app/src/main/resources/META-INF/flowable-app/flowable-app.properties]: Don't we want to allow debugging by default? as a feature to show.

File: modules/flowable-ui-task/flowable-ui-task-rest/src/main/java/org/flowable/app/rest/runtime/RuntimeDisplayJsonClientResource.java (hunk 1 above)
Comment (martin-grofcik) [reply modules/flowable-ui-task/flowable-ui-task-rest/src/main/java/org/flowable/app/rest/runtime/RuntimeDisplayJsonClientResource.java]: Done

**Comment by martin-grofcik** [comment]
@tijsrademakers: Please let me know the code review results.

**Comment by tijsrademakers** [comment]
@martin-grofcik two comments:

1.  Can you revert the following change?

-    protected List<AsyncRunnableExecutionExceptionHandler> customAsyncRunnableExecutionExceptionHandlers;
+    protected List<AsyncRunnableExecutionExceptionHandler> customAsyncRunnableExecutionExceptionHandlers = Collections.EMPTY_LIST;

2. In the Flowable task app UI grid is already part of the included libraries. UI Grid is the sucessor / new version of NG grid, so can use UI grid instead and remove the addition of the NG grid libraries?

**Pull request merged**